public/index.html

```diff
@@ -6,7 +6,51 @@
     <link rel="shortcut icon" href="favicon.png">
     <title>OpenWISP WiFi Login</title>
   </head>
+
+  <style>
+    #loading {
+    width: 60px;
+    height: 60px;
+    position: absolute;
+    top:50%;
+    left:50%;
+    }
+
+    #loading div {
+      box-sizing: border-box;
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      border: 8px solid transparent;
+      border-top-color: #708090;
+      border-radius: 50%;
+      animation: circle-one 1.2s linear infinite;
+    }
+    #loading div:nth-child(2) {
+      border: 8px solid transparent;
+      border-bottom-color: #708090;
+      animation: circle-two 1.2s linear infinite;
+    }
+
+    /* Loading Animation */
+    @keyframes circle-one {
+    0% { transform: rotate(0deg); border-width: 6px; }
+    50% { transform: rotate(180deg); border-width: 1px; }
+    100% { transform: rotate(360deg); border-width: 6px; }
+    }
+
+    @keyframes circle-two {
+      0% { transform: rotate(0deg); border-width: 1px; }
+      50% { transform: rotate(180deg); border-width: 6px; }
+      100% { transform: rotate(360deg); border-width: 1px; }
+    }
+  </style>
```

Comment:
we already have styles for the loader, is it possible to move these to the style tag so they can be reused across the whole application and avoid redefining the same styles multiple times?

See (except `.org-wrapper-not-found`):
https://github.com/openwisp/openwisp-wifi-login-pages/blob/50c2885200fe5b4369b119d0608bb03222eb5031/client/components/organization-wrapper/index.css#L17-L57

Reply:
Sir, one things i would like to ask that do we need to shift some of style tag code to index.css for their reusability? Is this what you are asking for?